### ISSUE TITLE
Zoom isn't limited to only ~/Documents/Zoom

### ISSUE THREAD

**Issue by FaruNL** [flathub/us.zoom.Zoom#226]
When sending a file or choosing a background, Zoom prompt me the file chooser, but the file chooser it's the normal one, not the restricted one:

**Normal file chooser:**
![Normal File Chooser](https://user-images.githubusercontent.com/34426099/97245370-86f07f80-17c0-11eb-8957-7b0033a6ecbe.png)
I can access all my disk through this.

**Restricted file chooser (OnlyOffice Flatpak example)**
![File Chooser with Restricted Access](https://user-images.githubusercontent.com/34426099/97245372-87891600-17c0-11eb-9e10-b0c8b95603c7.png)
It used to be like that a couple weeks ago, and it only allowed me the access to ~/Documents/Zoom.

I don't know if this happened due to a recent update, or what. Do you recommend me to roll back to another version, or is it something specific to my computer?

> If it's worth something: **OS: Kubuntu 20.04.1 LTS**

**Comment by TheEvilSkeleton** [comment]
Does the Zoom flatpak use the GTK file picker? If so, it means that it uses XDG file portals.

https://github.com/flatpak/flatpak/wiki/Portals

TL;DR:
> The file chooser portal provides indirect file access to sandboxed applications. The basic idea of the portal is that sandboxed apps are not allowed to directly open files (outside the filesystem ''playground'' that is part of the sandbox). Instead, the portal can be used to selectively make files available inside the sandbox

**Comment by FaruNL** [comment]
Yeah I was just reading that. I thought It was a problem since at the beginning it wasn't using the GTK file picker, but it doesn't seem to be a security threat, so I'm gonna close the issue, ty for the reply.
